pkg/auth/providers/oidc/oidc_actions.go

```diff
@@ -68,6 +72,12 @@ func (o *OpenIDCProvider) TestAndApply(request *types.APIContext) error {
 	}
 
 	oidcConfig = oidcConfigApplyInput.OIDCConfig
+	// set a default value for GroupSearchEnabled
+	// in case user input is nil for some reasons.
+	if oidcConfigApplyInput.OIDCConfig.GroupSearchEnabled == nil {
+		falseBool := false
+		oidcConfig.GroupSearchEnabled = &falseBool
```

Comment:
Also writable as `pointer.BoolPtr(false)`, without a helper variable.
Requires import of `"k8s.io/utils/pointer"`